client.py

```diff
@@ -87,8 +87,11 @@ def register(self):
         if 'client_id' in self.config:
             raise Exception('Client is already registered')
 
-        if 'dcr_access_token' not in self.config:
```

Comment:
The idea here was to allow to configure a static access token, not sure there is a case for it though.

Reply:
The DCR tokens are nonces, so that won't work